code/modules/hivemind/wires.dm

```diff
@@ -0,0 +1,290 @@
+//Wireweeds is an actually wires that covered with metal shell.
+//They are grow and spread under rogue AI control and with help of unknown, dangerous nano-tech
+//When they reach any machine, they annihilate them and rebuild into something horrible. This is 'hands' of our rogue AI
+
+/obj/effect/plant/hivemind
+	layer = 2
+	health = 80
+	max_health = 80 //we are a little bit durable
+	var/list/killer_reagents = list("pacid", "sacid", "hclacid", "thermite")
+	//internals
+	var/obj/structure/hivemind_machine/node/master_node
+	var/list/wires_connections = list("0", "0", "0", "0")
+
+
+/obj/effect/plant/hivemind/New()
+	..()
+	icon = 'icons/obj/hivemind.dmi'
+	spawn(2)
+		update_neighbors()
+
+
+/obj/effect/plant/hivemind/Destroy()
+	if(master_node)
+		master_node.my_wireweeds.Remove(src)
+	return ..()
+
+
+/obj/effect/plant/hivemind/after_spread(obj/effect/plant/child, turf/target_turf)
+	if(master_node)
+		master_node.add_wireweed(child)
+	spawn(1)
+		child.dir = get_dir(loc, target_turf) //actually this means nothing for wires, but need for animation
+		flick("spread_anim", child)
+		child.forceMove(target_turf)
+		update_icon()
+
+
+/obj/effect/plant/hivemind/proc/try_to_annihilate()
+	if(hive_mind_ai && master_node)
+		for(var/obj/machinery/machine_on_my_tile in loc)
+			var/can_annihilate = TRUE
+
+			//whitelist check
+			if(is_type_in_list(machine_on_my_tile, hive_mind_ai.restricted_machineries))
+				can_annihilate = FALSE
+
+			//annihilation is slow process, so it's take some time
+			//there we use our failure chance. Then it lower, then faster hivemind learn how to properly annihilate it
+			if(can_annihilate && prob(hive_mind_ai.failure_chance))
+				can_annihilate = FALSE
+				shake_the(machine_on_my_tile)
+				return
+
+			 //only one machine per turf
+			if(can_annihilate && !locate(/obj/structure/hivemind_machine) in loc)
+				annihilate(machine_on_my_tile)
+			//other will be... merged
+			else if(can_annihilate)
+				qdel(machine_on_my_tile)
+
+		//modular computers handling
+		var/obj/item/modular_computer/console/mod_comp = locate() in loc
+		if(mod_comp)
+			annihilate(mod_comp)
+
+		//dead bodies handling
+		for(var/mob/living/dead_body in loc)
+			if(dead_body.stat == DEAD)
+				annihilate(dead_body)
+
+
+/obj/effect/plant/hivemind/update_neighbors()
+	..()
+	update_connections()
+	update_icon()
+	update_openspace()
+
+
+/obj/effect/plant/hivemind/spread()
+	if(hive_mind_ai && master_node)
+		..()
+
+
+/obj/effect/plant/hivemind/life()
+	if(hive_mind_ai && master_node)
+		try_to_annihilate()
+		chem_handler()
+	else
+		//slow vanishing after node death
+		health -= 10
+		alpha = 255 * health/max_health
+		check_health()
+
+
+/obj/effect/plant/hivemind/is_mature()
```

Comment:
I recall in the last test, the wires were instantly at full maturity when spawned, this may be why.

They should grow gradually to make use of their cool sprites